schema/swagger-extensions.json

```diff
@@ -1589,6 +1592,51 @@
         "enum": ["create", "read", "update"]
       },
       "uniqueItems": true
+    },
+    "xmsResponseExample": {
+      "type": "object",
+      "description": "Provides the format of specifying an example response in the x-ms-examples extension.",
+      "properties": {
+        "headers": {
+          "type": "object",
+          "description": "Examples of response headers like 'location', 'azure-asyncoperation', 'retry-after', etc.",
```

Comment:
>'location', 'azure-asyncoperation', 'retry-after', etc.", 

Examples should be in the canonical HTTP casing: 'Location', 'Azure-AsyncOperation', 'Retry-After'